.github/workflows/ci.yaml

```diff
@@ -763,33 +763,51 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        test:
-          - TestVersion
-          - TestCommandsRequireSudo
-          - TestResetAndReinstallAirgap
-          - TestSingleNodeAirgapUpgrade
-          - TestSingleNodeAirgapUpgradeConfigValues
-          - TestSingleNodeAirgapUpgradeCustomCIDR
-          - TestSingleNodeDisasterRecoveryWithProxy
-          - TestProxiedEnvironment
-          - TestProxiedCustomCIDR
-          - TestInstallWithPrivateCAs
-          - TestInstallWithMITMProxy
         include:
-          - test: TestMultiNodeAirgapUpgrade
-            runner: embedded-cluster
-          - test: TestMultiNodeAirgapUpgradeSameK0s
-            runner: embedded-cluster
-          - test: TestMultiNodeAirgapUpgradePreviousStable
-            runner: embedded-cluster
-          - test: TestAirgapUpgradeFromEC18
-            runner: embedded-cluster
-          - test: TestSingleNodeAirgapDisasterRecovery
-            runner: embedded-cluster
-          - test: TestMultiNodeAirgapHAInstallation
-            runner: embedded-cluster
-          - test: TestMultiNodeAirgapHADisasterRecovery
+          - test: TestFiveNodesAirgapUpgrade
```

Comment:
This looks like the reverse of what you want

Reply:
Just need to revert the "test" commit

Reply:
it is evidently still running the 5 node test on this PR